samples/csharp/MicrosoftEduGraphSamples/Workflows/SubmissionWorkflow.cs

```diff
@@ -35,21 +36,21 @@ public async Task ReassignWorkflow()
                 string submissionId = string.Empty;
 
                 // Get a Graph client using delegated permissions
-                var graphClient = MicrosoftGraphSDK.GraphClient.GetDelegateClient(_config["tenantId"], _config["appId"], _config["teacherAccount"], _config["password"]);
+                var graphClient = GraphClient.GetDelegateClient(_config["tenantId"], _config["appId"], _config["teacherAccount"], _config["password"]);
 
                 // Teacher creates a new assignment
-                var assignment = await MicrosoftGraphSDK.Assignment.CreateAsync(graphClient, _config["classId"]);
+                var assignment = await Assignment.CreateAsync(graphClient, _config["classId"]);
```

Comment:
similar comment: either rename it to CreateSampleAsync or pass in request body

Reply:
renamed "CreateAsync" to "CreateSampleAsync" method